pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -60,12 +60,14 @@ const (
 	IstioInjectAnnotation = "sidecar.istio.io/inject"
 )
 
-type BindMechanism interface {
+type PodNetworkInfraConfigurator interface {
 	discoverPodNetworkInterface(podIfaceName string) error
 	preparePodNetworkInterface() error
 	generateDomainIfaceSpec() api.Interface
 	generateDHCPConfig() *cache.DHCPConfig
+}
 
+type BindMechanism interface {
```

Comment:
I agree with Radim's comment. It is marked as resolved but I still see the old name.

Reply:
Check out the next commit :)